hack/aks/property-based-scheduling.md

```diff
@@ -0,0 +1,456 @@
+# az account set --subscription YOUR-SUBSCRIPTION-ID
+
+# Replace eastus with any other Azure location, if necessary.
+export RG=fleet-demo
+export LOCATION_1=eastus
+# Create a resource group.
+az group create -n $RG -l $LOCATION_1
+
+# Create the hub cluster.
+export HUB_CLUSTER=hub
+az aks create -n $HUB_CLUSTER \
+    -l $LOCATION_1 \
+    -g $RG \
+    --node-count 2
+```
+
+It may take a few moments before the commands complete.
+
+Next, create the three member clusters using the commands below:
+
+```sh
+export MEMBER_CLUSTER_1=bravelion
+export MEMBER_CLUSTER_2=smartfish
+export MEMBER_CLUSTER_3=jumpingcat
+
+# Similarly, replace the location values with ones of your own, if necessary.
+export LOCATION_2=centralus
+export LOCATION_3=westus
+
+# Create the three member clusters.
+az aks create -n $MEMBER_CLUSTER_1 \
+    -l $LOCATION_1 \
+    -g $RG \
+    --node-count 2 
+
+az aks create -n $MEMBER_CLUSTER_2 \
+    -l $LOCATION_2 \
+    -g $RG \
+    --node-count 3
+
+az aks create -n $MEMBER_CLUSTER_3 \
+    -l $LOCATION_3 \
+    -g $RG \
+    --node-count 4 
+```
+
+It may take a few moments before the command completes.
+
+After all the commands have completed successfully, use the command below to retrieve their
+credentials for later access:
+
+```sh
+az aks get-credentials -n $HUB_CLUSTER -g $RG --admin
+az aks get-credentials -n $MEMBER_CLUSTER_1 -g $RG --admin
+az aks get-credentials -n $MEMBER_CLUSTER_2 -g $RG --admin
+az aks get-credentials -n $MEMBER_CLUSTER_3 -g $RG --admin
+```
+
+## Deploy Fleet
+
+To set up Fleet using the clusters you just created, you will need to install Fleet hub and
+member agents on the respective clusters.
+
+First, clone the Fleet source code repository, which contains the Helm charts used for Fleet
+agent installation:
+
+```sh
+git clone https://github.com/Azure/fleet.git
+cd fleet
+git checkout demo
+```
+
+Then install the Fleet hub agent in the Fleet hub cluster:
+
+```sh
+kubectl config use-context $HUB_CLUSTER-admin
+
+export REGISTRY=fleetdemo.azurecr.io
+export TAG=demo
+helm install hub-agent charts/hub-agent/ \
+    --set image.pullPolicy=Always \
+    --set image.repository=$REGISTRY/hub-agent \
+    --set image.tag=$TAG \
+    --set logVerbosity=2 \
+    --set namespace=fleet-system \
+    --set enableWebhook=true \
+    --set webhookClientConnectionType=service \
+    --set enableV1Alpha1APIs=false \
+    --set enableV1Beta1APIs=true
+```
+
+It will take a few moments to complete the installation. After the command returns, verify that the
+Fleet hub agent is up and running with this command:
+
+```sh
+kubectl get pods -n fleet-system
+```
+
+You should see all pods in the ready state. Note that the hub agent may take a few seconds
+to get ready after the installation.
+
+Next, install the Fleet member agents in the Fleet member clusters. The member agent needs
+their credentials set up in the hub cluster so that they can access the hub cluster; see the
+commands below for details.
```

Comment:
```suggestion
You should see all pods in the ready state. Note that the hub agent may take a moment to get ready after the installation.

Next, install the Fleet member agents in the Fleet member clusters. The member agent needs their credentials set up in the hub cluster so that they can access the hub cluster; see the commands below for details.
```